package.json

```diff
@@ -30,7 +30,9 @@
     "test": "jest --coverage",
     "size": "yarn build && size-limit",
     "storybook:build": "build-storybook -o .storybook/static/storybook",
-    "heroku-postbuild": "yarn run storybook:build"
+    "heroku-postbuild": "yarn run storybook:build",
+    "format": "prettier --write \"src/**/*.+(js|jsx|json|scss)\"",
+    "format-storybook": "prettier --write \".storybook/**/*.+(js|jsx|json|scss)\""
```

Comment:
I know we don't normally include styles, but is there a reason for that?

Reply:
This was a remnant of not wanting to interfere with how the design team does their work. I think we can update it